code/datums/flock/flock.dm

```diff
@@ -758,33 +786,99 @@ proc/get_default_flock()
 			available_tiles -= src.busy_tiles[owner]
 		return available_tiles
 
-// PROCESS
-
+/// Process to check what to do with the relay based off compute/tiles, specifically for before placement
 /datum/flock/proc/relay_process()
-	if (src.total_compute() > 300)
+
+	if (src.flockmind?.tutorial || src.relay_finished)
+		return
+
+	// Handle relay post-placement
+	if (src.relay_in_progress)
+		src.time_left = src.last_relay.get_time_left()
+		return
+
+	var/total_tiles = length(src.all_owned_tiles)
+	// A value from 0 to 1 representing progress towards creating the Relay.
+	var/pct_to_relay = min(total_tiles / FLOCK_RELAY_TILE_REQUIREMENT, 1) * min(src.total_compute() / FLOCK_RELAY_COMPUTE_COST, 1)
```

Comment:
Maybe averaging these would make more sense than multiplying? Otherwise you're going to get very low values most of the time.

Reply:
mmm tru